topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md

```diff
@@ -390,6 +411,12 @@ A new dataset will be created in your history, containing the JBrowse interactiv
 
 You can now click on the names of the tracks to add them in, try the vcf file and gff file. You can see where the variants are located and which genes they are in. If you click on the BAM file you can zoom right in to see the read alignments for each variant if you wish.
```

Comment:
you could consider adding the data/ folder and then including a snippet like we do in https://training.galaxyproject.org/training-material/topics/assembly/tutorials/chloroplast-assembly/tutorial.html#hands-on-visualise-mapped-reads (just after that hands on)

Reply:
Sounds good. Where do you get that data from - the `trackList.json` etc? By downloading an existing JBrowse dataset?